publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/ProducerController.java

```diff
@@ -303,7 +306,14 @@ public ResponseEntity generateAndPublish(final String msgProtocol, final String
         if (bodyJson.isJsonObject()) {
             events.add(getAsJsonObject(bodyJson));
         } else if (bodyJson.isJsonArray()) {
-            for (JsonElement element : bodyJson.getAsJsonArray()) {
+            JsonArray bodyJsonArray = bodyJson.getAsJsonArray();
+            //here add check for limitation for events in array is fetched from REMReM property and checked during publishing.
+            if (bodyJsonArray.size() > maxSizeOfInputArray) {
+                return createResponseEntity(HttpStatus.BAD_REQUEST, JSON_ERROR_STATUS,
+                        "The number of events in the input array is too high: " + bodyJsonArray.size() + " > " + maxSizeOfInputArray + ", " +
```

Comment:
-> maxSizeOfInputArray + "; " + "you ... to increase it."

Reply:
Done